blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/preload/OrganizationStatePreloader.java

```diff
@@ -44,4 +66,26 @@ public String getStateJson() {
             return "{}";// if will happen only when there is no implementation of BlueOrganization found.
         }
     }
+
+    private static final Pattern pattern = Pattern.compile("\\/organizations\\/([^\\/]*)\\/");
```

Comment:
should this not match `/blue/organisations/([^/]+)/`  otherwise if someone deployed their jenkins to "orgainizations" in the context it would fail (just narrows the window of things going wrong, does not completely eliminate it.